pkg/bindings/test/test_test.go

```diff
@@ -7,13 +7,14 @@ import (
 	"github.com/stretchr/testify/assert"
 
 	"github.com/cloudevents/sdk-go/pkg/binding"
-	"github.com/cloudevents/sdk-go/pkg/binding/test"
+	. "github.com/cloudevents/sdk-go/pkg/binding/test"
+	. "github.com/cloudevents/sdk-go/pkg/bindings/test"
 )
 
 func TestEvent(t *testing.T) {
```

Comment:
IMO This should continue to live in `"github.com/cloudevents/sdk-go/pkg/binding/test"`

Reply:
These move in the next PR, I do a split